internal/olm/client/client.go

```diff
@@ -280,10 +280,10 @@ func (c Client) printPodErrors(ctx context.Context, depSelectors *metav1.LabelSe
 		return fmt.Errorf("error getting Pods: %v", err)
```

Comment:
Could you please add the new changelog fragment in changelog/fragments (see changelog/fragments/00-template.yaml) for we are able to get it merged?